src/pages/ModelsIndex/_models.scss

```diff
@@ -5,11 +5,20 @@
   &__header {
     align-items: center;
     display: grid;
+    gap: 1rem;
```

Comment:
This gap was moved out of the media query so that it adds space between the segmented control and the search box on mobile.

Reply:
That's a nice touch! I was wondering if maybe a `0.5rem` gap would be more suitable, considering that this way we would have a more or less similar distance between it and the `Group by` block above it and the horizontal separator bellow it?

Reply:
Good point, fixed!